package-lock.json

```diff
@@ -1546,6 +1547,16 @@
             "resolved": "https://registry.npmjs.org/argparse/-/argparse-2.0.1.tgz",
             "integrity": "sha512-8+9WqebbFzpX9OR+Wa6O29asIogeRMzcGtAINdpMHHyAg10f05aSFVBbcEqGf/PXw1EjAZ+q2/bEBg3DvurK3Q=="
         },
+        "node_modules/azure-devops-node-api": {
+            "version": "13.0.0",
+            "resolved": "https://devdiv.pkgs.visualstudio.com/_packaging/Intellicode/npm/registry/azure-devops-node-api/-/azure-devops-node-api-13.0.0.tgz",
```

Comment:
You seem to have committed to the repo with a bad npmrc.

Reply:
Fixed.